apps/src/templates/CollapsibleSection.tsx

```diff
@@ -78,7 +79,19 @@ const CollapsibleSection: React.FunctionComponent<CollapsibleSectionProps> = ({
           </Typography>
         </button>
       </div>
-      {!collapsed && children}
+      {/* The tradeoff for the nice animation here is that everything gets loaded even if collapsed. 
+      We might not want that on every usage of this? I think it's pretty neat tho.? */}
+      <div
```

Comment:
The tradeoff for the nice animation here is that everything gets loaded even if collapsed. We might not want that on every usage of this, if any usages are meant to make page load faster? I think it's pretty neat tho.

Reply:
I think the animated collapse/expand is super cool! 

Tagging @sanchitmalhotra126 who first created this component to see what he thinks. 

One thing in terms of a11y is that if a user tab-navigates from the dropdown label while collapsed, they'll navigate to the link in the expanded content, but won't see the link highlighted on the screen.
Also - may want to do a quick check with Sam/Mark to see if they're good with this default behavior.

Reply:
Going to leave this on the back burner for a bit while I am helping out with gen ai stuff this sprint, but my plan is to try out the aria properties described here https://accessibility.huit.harvard.edu/technique-expandable-sections and see if I can make it accessible with the animation. If not, I'll remove the animation part and opt for accessibility.

Reply:
Thanks Cassi for the update!